docs/source/api/plugin/landing-pages.mdx

```diff
@@ -153,6 +153,28 @@ An object containing initial HTTP header values to populate in the Explorer on l
 <tr>
 <td>
 
+##### `collectionId` & `operationId`
+
+`string`
+
+</td>
+<td>
+
+The ID of a collection, paired with an operation ID to populate in the Sandbox on load. You can find these values from a registered graph in Studio by clicking the **...** menu next to an operation in the Explorer of that graph and selecting **View operation details**.
+
+Example:
+
+```jsx
+collectionId: 'abc1234',
+operationId: 'xyz1234'
+```
```

Comment:
The examples don't look super in the table but I like the idea. In this case I think it's a bit trivial / not worth providing an example since they don't present that well.

Probably worth calling out explicitly that both must be provided (I do like that they're grouped for that reason).

Update further down with the same.

Reply:
https://deploy-preview-7445--apollo-server-docs.netlify.app/api/plugin/landing-pages#collectionid--operationid

Reply:
Should be fine, the description does say "paired with".